src/mobile/screens/SignInScreen/SignInScreen.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  TextInput,
+} from 'react-native';
+import Relay from 'react-relay';
+import { Actions } from 'react-native-router-flux';
+import {
+  mainTextColor,
+  normalTextSize,
+  greyColor,
+  errorColor,
+  linkColor,
+} from '../../config/StyleConst';
+import AppRoutes from '../../AppRoutes';
+import SignInMutation from '../../mutations/SignInMutation';
+import { setToken } from '../../services/AuthorizationService';
+
+const styles = StyleSheet.create({
+  container: {
+    flexGrow: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: 10
+  },
+  welcome: {
+    fontSize: normalTextSize,
+    textAlign: 'center',
+    margin: 10,
+    color: mainTextColor,
+  },
+  emailInput: {
+    height: 40,
+    borderColor: greyColor,
+    borderWidth: 1,
+    marginBottom: 10,
+  },
+  passwordInput: {
```

Comment:
Some of the styles are exactly the same (like this one and `emailInput`). Can we just reuse them?